list_reports.R

```diff
@@ -5,17 +5,27 @@ library(stringr)
 
 source("./read_config.R")
 
+# file: The filename of the SeqWare TSV Run Report
+# path: The path to the SeqWare TSV file
+# dir_path: The path to the directory holding the SeqWare TSV file
+# name: The human readable name that is exposer to user
 listRunReports <- function() {
-  path <- list.files(normalizePath(CONFIG.RUNPATH), pattern = "*.tsv$", full.names = TRUE)
-  file <- sapply(path, basename)
+  path <- list.files(normalizePath(CONFIG.RUNPATH), pattern = ".*\\.tsv$", full.names = TRUE, recursive = TRUE)
+  file <- basename(path)
   name <- str_match(file, "(.*?)_report.tsv")[, 2]
   dt <- data.table(path = path,
+                   dir_path = dirname(path),
                    file = file,
                    name = name)
   
+  # This is temporary code to exclude the TSV files in the base directory from eariler app version
```

Comment:
When will the temporary code be removed? Is it a matter of updating the cron job to no longer dump the TSV files into the base directory, or does more need to happen?

Reply:
You are correct. Once the cron job no longer puts TSV files into the base directory, this code can be deleted. I will put in a ticket to modify the cron job once this Pull Request is merged into production.